### PR TITLE
fix(sdk-core): tss tx signing

### PR DESCRIPTION
**Pull request by olibeyene**
Ticket: BG-56442

### CODE REVIEW

**Review by wesgraham — DISMISSED** [review]

File: modules/sdk-coin-eth/src/eth.ts [hunk 1]
```diff
@@ -1618,7 +1629,12 @@ export class Eth extends BaseCoin {
    */
   async verifyTransaction(params: VerifyEthTransactionOptions): Promise<boolean> {
     const ethNetwork = this.getNetwork();
-    const { txParams, txPrebuild, wallet } = params;
+    const { txParams, txPrebuild, wallet, walletType } = params;
+
+    if (walletType === 'tss') {
```
Comment (byhow) [inline modules/sdk-coin-eth/src/eth.ts:1634]: nit: use an enum for wallet type

File: modules/sdk-core/src/bitgo/baseCoin/iBaseCoin.ts [hunk 2]
```diff
@@ -140,6 +140,7 @@ export interface VerifyTransactionOptions {
   wallet: IWallet;
   verification?: VerificationOptions;
   reqId?: IRequestTracer;
+  walletType?: 'onchain' | 'tss';
```
Comment (byhow) [inline modules/sdk-core/src/bitgo/baseCoin/iBaseCoin.ts:143]: ditto

File: modules/sdk-core/src/bitgo/tss/common.ts [hunk 3]
```diff
@@ -37,10 +37,16 @@ export async function sendSignatureShare(
   walletId: string,
   txRequestId: string,
   signatureShare: SignatureShareRecord,
-  signerShare?: string
+  signerShare?: string,
+  mpcAlgorithm: 'eddsa' | 'ecdsa' = 'eddsa'
```
Comment (byhow) [inline modules/sdk-core/src/bitgo/tss/common.ts:41]: ditto

**Review by byhow — APPROVED** [review]
just a nit but can be resolved separately

**Pull request merged**